deploy/helm/listener-operator/values.yaml

```diff
@@ -89,3 +89,9 @@ affinity: {}
 
 # Kubelet dir may vary in environments such as microk8s, see https://github.com/stackabletech/secret-operator/issues/229
 kubeletDir: /var/lib/kubelet
+
+# Options: none, vanilla, premium
+# none: No ListenerClasses are preinstalled, the administrator must supply them themself
+# pets: ListenerClasses are preinstalled that are suitable for on-prem/"pet" environments, assuming long-running Nodes but not requiring a LoadBalancer controller
+# cattle: ListenerClasses are preinstalled that are suitable for cloud/"cattle" environments, requiring a LoadBalancer controller but tolerating short-lived Nodes
```

Comment:
How about `pet-nodes`/`cattle-nodes`?

I, personally, wouldn't call on-prem k8s "cloud".

> Additionally, I would not name my dog "stackable-data-platform-worker-1".

![image](https://github.com/stackabletech/listener-operator/assets/649832/eb61a70f-4214-4818-a48b-c6e4961ba52e)

:P

Reply:
While I personally don't have any issue with the pets/cattle analogy in principle a quick survey here in the office has shown it to be not commonly known.

Maybe something like "stable-nodes" and "ephemeral-nodes" could work as a compromise?

Also, one brief question, what would we default to if the user doesn't specify anything?

Reply:
> what would we default to if the user doesn't specify anything?

The default preset is `pets`, so that it works out of the box on a vanilla Kubernetes.

Reply:
While I personally don't have any issue with the pets/cattle analogy in principle a quick survey here in the office has shown it to be not commonly known.

Maybe something like "stable-nodes" and "ephemeral-nodes" could work as a compromise?

Also, one brief question, what would we default to if the user doesn't specify anything? I seem to be too dumb to properly read the helm chart..